.github/workflows/weekly_check.py

```diff
@@ -0,0 +1,18 @@
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--fileNameList', type=str, default='using_default_location')
+    args = parser.parse_args()
+    name_list = args.fileNameList.split(":")
+    folder_need_check = []
+    for loc in name_list:
+        # Avoid '' sometimes caused by changedFile action.
+        if loc != '' and loc != 'README.md':
```

Comment:
In what case `loc` will be a empty string?

Reply:
This is not an elegant way of writing since you manually check for files like `README.md`. What if some extra files are put under the `examples/` directory besides `README.md`？

Reply:
changedFileName=""
            for file in ${{ steps.changed-files.outputs.all_changed_files  }}; do
              changedFileName="${file}:${changedFileName}"
            done
  Here will introduce a ''. Either use if in shell or use if in python. I guess using it in python is easier? Thanks!

Reply:
My second comment is not handled.